### PR TITLE
Use push.registry.devshift.net

### PR DESCRIPTION
**Pull request by vpavlin**

### CODE REVIEW

**Comment by vpavlin** [comment]
@aslakknutsen ping

**Review by aslakknutsen — APPROVED** [review]

**Comment by aslakknutsen** [comment]
[test]

File: Makefile [hunk 1]
```diff
@@ -58,15 +58,23 @@ docker-build-run:
 
 .PHONY: docker-run-deploy
 docker-run-deploy:
-	[ -f jenkins-env ] && cat jenkins-env | grep GIT > inherit-env
+	[ -f jenkins-env ] && cat jenkins-env | grep -e GIT -e DEVSHIFT > inherit-env
 	[ -f inherit-env ] && . inherit-env
+	REGISTRY="push.registry.devshift.net"
```
Comment (maxandersen) [inline Makefile:63]: educate me here: what is the diff between `push.registry.devshift.net` and `registry.devshift.net` ?

Comment (aslakknutsen) [reply Makefile:63]: Clearly, it is one URL to push to and one to read from.. .. ... .. .. . . .

**Pull request merged**